docs/pipelines/cpu-to-carbon.md

```diff
@@ -0,0 +1,359 @@
+| `x`, `y`                | Points on power curve relating CPU utilization to a coefficient used to scale the processor's thermal design power | `x: [0, 10, 50, 100], y: [0.12, 0.32, 0.75, 1.02]` | dimensionless | [Davy, 2021](https://medium.com/teads-engineering/building-an-aws-ec2-carbon-emissions-dataset-3f0fd76c98ac) |
+| `grid-carbon-intensity` | the carbon emitted per unit energy from the electrical grid                                                        | 750                                                | gCO2e/kWh     | approximates global average                                                                                  |
+
+
+## Assumptions and limitations
+
+The following are assumed to be true in this manifest:
+
+- the power curve relating CPU utilization to power is appropriate for the processor being used to run our application
+- the temporal granularity of the observations are sufficient to accurately capture the behaviour of our application
+- the grid carbon intensity is sufficiently accurate for the location where the computational work is done
+
+
+## Components
+
+There is only one component in this example. It represents the entire application. The component pipeline looks as follows:
+
+```yaml
+pipeline:
+  compute:
+    - interpolate
+    - cpu-factor-to-wattage
+    - wattage-times-duration
+    - wattage-to-energy-kwh
+    - calculate-vcpu-ratio
+    - correct-cpu-energy-for-vcpu-ratio
+    - energy-to-carbon
+```
+
+
+## Plugins
+
+### Interpolate
+
+The `interpolate` plugin is used once. The instance is named `interpolate`. It is used to interpolate the curve relating CPU utilization and thermal-design-power factor so that the right value canb be retrived for the observed CPU utilization at each timestep.
+
+#### config
+
+```
+method: linear
+x: [0, 10, 50, 100]
+y:[0.12, 0.32, 0.75, 1.02]
+input-parameter: cpu/utilization
+output-parameter: cpu-factor
+```
+
+### Multiply
+
+The `Multiply` plugin is used several times. The instances are:
+
+- `cpu-factor-to-wattage`: used to multiply the thermal design power of the processor by the factor returned from the power curve interpolation, yielding power in Watts.
+- `wattage-times-duration`: used to multiply the power in Watts by the duration of each timestep, yielding energy in W/duration.
+- `energy-to-carbon`: used to convert energy expended to carbon emitted.
+
+#### config
+
+```
+cpu-factor-to-wattage:
+input-parameters:
+  - cpu-factor
+  - cpu/thermal-design-power
+output-parameter: 
+  - cpu-wattage
+
+wattage-times-duration:
+input-parameters:
+  - cpu-wattage
+  - duration
+output-parameter: 
+  - cpu-wattage-times-duration
+
+energy-to-carbon:
+input-parameters:
+  - grid-carbon-intensity
+  - energy-cpu-kwh
+output-parameter: 
+  - carbon
+
+```
+
+### Divide
+
+The `Divide` plugin is used once in this manifest. The instance is named `wattage-to-energy-kwh`. It is used to convert energy in W/duration to kWh.
```

Comment:
Although it is written that the Divide plugin is used once, in the example manifest below it is used 3 times

Reply:
thanks for catching this! Fixing now